### PR TITLE
TOMEE-2873 Translate to Portuguese: examples/mp-rest-jwt-jwk

### PR DESCRIPTION
**Pull request by Daniel-Dos**
Link to Jira: https://issues.apache.org/jira/browse/TOMEE-2873

### CODE REVIEW

File: examples/mp-rest-jwt-jwk/README_pt.adoc [hunk 1]
```diff
@@ -0,0 +1,69 @@
...
+  "preferred_username": "john",
+  "groups": [
+    "guest", "admin"
+  ]
+}
+----
+
+== Acesse os endpoints com JWT token
+
+Acessamos pontos de extremidade de nossa classe de teste criando um `JWT` com a ajuda do nosso `TokenUtils.generateJWTString (String jsonResource, String keyId)` que assina nossos dados de usuário no formato json com a ajuda do nosso private key `src/test/resources/{keyId}` .
```
Comment (willesreis) [inline examples/mp-rest-jwt-jwk/README_pt.adoc:67]: Here, the expression *endpoints* sounds better than *pontos de extremidade* and keep similar to subtitle above.

File: examples/mp-rest-jwt-jwk/README_pt.adoc [hunk 2]
```diff
@@ -0,0 +1,69 @@
+= MicroProfile JWT JWKs
+:index-group: MicroProfile
+:jbake-type: page
+:jbake-status: published
+ 
+Este é um exemplo de como usar MicroProfile JWT em TomEE usando  a public ke como JWKs.
```
[the remark this answers is not in the document]
Comment (Daniel-Dos) [reply examples/mp-rest-jwt-jwk/README_pt.adoc:6]: I left it in English.

**Comment by Daniel-Dos** [comment]
thanks @willesreis to review.

**Comment by cesarhernandezgt** [comment]
+1 thanks @Daniel-Dos and @willesreis.

**Pull request merged**